deployment/src/main/java/io/quarkiverse/loggingmanager/deployment/LoggingManagerProcessor.java

```diff
@@ -32,18 +32,18 @@
 import io.vertx.core.Handler;
 import io.vertx.ext.web.RoutingContext;
 
-class LoggingUiProcessor {
-    private static final String FEATURE = "quarkus-logging-ui";
+class LoggingManagerProcessor {
+    private static final String FEATURE = "quarkus-logging-manager";
```

Comment:
```suggestion
    private static final String FEATURE = "logging-manager";
```
to be consistent with the other features.